acceptance_tests/features/steps/view_collection_exercise_details.py

```diff
@@ -4,10 +4,15 @@
 
 
 @given('the 1803 collection exercise for the QBS survey has been created')
-def bres_2017_exists(_):
+def qbs_1803_exists(_):
     pass
 
 
+@given('the 1803 collection exercise events for the QBS survey has been created')
+def bres_2017_events_exist(_):
+    pass
```

Comment:
how are the events being loaded? if i run these tests on my mac the test will fail?

Reply:
I was following the previous pattern we'd used in these tests so you have to run the tests with the assumption that surveys/collection_exercises/events are already in the system.

Would probably be good to have a local version that did the set up for this on local machines or did them through posting to endpoints during the tests

Reply:
This would depend on the story itself if it's the same as the collection exercise data being loaded then the tests would be the same, and the data being loaded would be an environmental setup not a test setup.

Reply:
Have we solved the way the application is going to load the data? I am concerned that without that being done tests are going to fail

Reply:
Vibhor and I are working on a way of loading data for another test once we have figured this out we will communicate on how data will be loaded.